research/conditional/utils/conditional_trainer.py

```diff
@@ -179,7 +204,33 @@ def train(self, n_steps: int):
                     except:
                         print("Decoding failed, skipping...")
                 self._after_step_operations(step)
-        self._after_train_operations()
+                if self.scheduler_trapezoidal_slides:
+                    for e in self.scheduler_trapezoidal_slides:
+                        if step == e["split_step"]:
+                            splitted_loggers = None
+                            if self.is_logging_process:
+                                splitted_loggers = [self.logger.loggers[0]]
+                                del self.logger.loggers[0]
```

Comment:
Let's do `stop` or something instead of `del`.

Reply:
https://docs.neptune.ai/api/run/#stop

Reply:
![image](https://github.com/user-attachments/assets/1e4f8169-88b5-496c-bd7f-fb0434ca310b)
What do you think?

Reply:
k, added for caution stop method call for neptune runs detaching from slurm run in mid training